tests/ui/elements/list.ts

```diff
@@ -200,6 +200,21 @@ export default class List extends CustomElement {
 
     return listItem.getAttribute(attribute);
   }
+
+  /**
+   * Counts the number of items in the list
+   */
+  async count(): Promise<number> {
+    await this.getListItem(0); // wait for first element to be visible
```

Comment:
Could you reuse here the isListItemVisible or isListNotEmpty() method? It will work a bit faster in this way.